sdks/python/apache_beam/ml/inference/sklearn_inference_test.py

```diff
@@ -62,6 +83,44 @@ def build_model():
   return model
 
 
+def pandas_dataframe():
+  csv_string = (
+      'category_1,number_1,category_2,number_2,label,number_3\n'
```

Comment:
Does this implicitly add indexes starting from 0? What if the user has custom indexes?